<launch>
  <include file="$(find ric_board)/launch/hector_slam.launch">
      <arg name="tf_map_scanmatch_transform_frame_name" value="scanmatcher_frame"/>
      <arg name="base_frame" value="base_link"/>
      <arg name="odom_frame" value="odom_link"/>
      <arg name="map_frame" value="/map"/>
      <arg default="true" name="pub_map_odom_transform"/>
      <arg default="5" name="scan_subscriber_queue_size"/>
      <arg name="scan_topic" value="/komodo_1/scan"/>
      <arg default="2048" name="map_size"/>
    </include> 

   <include file="$(find komodo_2dnav)/launch/move_base.launch"/>
</launch>
